internals/scripts/helpers/get-npm-config.js

```diff
@@ -0,0 +1,3 @@
+const fs = require('fs');
+
+module.exports = JSON.parse(fs.readFileSync('package.json', 'utf8'));
```

Comment:
Merge conflict issue. This file shouldn't be here.

Reply:
ugh not sure how that happened